Makefile

```
@@ -90,3 +90,53 @@ set-version:
 docs: # Build html docs
 	rm -f docs/external/*.md
 	cd docs && make html
+
+.PHONY: coverage
+coverage:
+	find storey -name '*.pyc' -exec rm {} \;
+	find tests -name '*.pyc' -exec rm {} \;
+	rm -rf coverage_reports;
+	rm -f unit_tests.coverage
+	COVERAGE_FILE=unit_tests.coverage coverage run --rcfile=unit_tests.coveragerc --source=. -m pytest --ignore=integration -rf -v;
+	coverage report
+	coverage xml -o coverage_reports/coverage_unit_tests.xml
+
+
+.PHONY: full-coverage-unit-tests
+full-coverage-unit-tests:
+	find storey -name '*.pyc' -exec rm {} \;
+	find tests -name '*.pyc' -exec rm {} \;
+	find integration -name '*.pyc' -exec rm {} \;
+	rm -rf coverage_reports;
+	rm -f full_unit_tests.coverage;
+	COVERAGE_FILE=full_unit_tests.coverage coverage run --rcfile=integration_tests.coveragerc --source=. -m pytest --ignore=integration -rf -v;
+	echo "coverage unit test report without excluding integration files:";
+	COVERAGE_FILE=full_unit_tests.coverage coverage report --rcfile=integration_tests.coveragerc;
+
+.PHONY: coverage-integration
+coverage-integration:
+	find storey -name '*.pyc' -exec rm {} \;
+	find tests -name '*.pyc' -exec rm {} \;
+	find integration -name '*.pyc' -exec rm {} \;
+	rm -rf coverage_reports;
+	rm -f integration.coverage;
+	COVERAGE_FILE=integration.coverage coverage run --rcfile=integration_tests.coveragerc  --source=. -m pytest -rf -v integration
+	echo "coverage integration report:";
```

```suggestion
	@echo Coverage integration report:;
```
Applies everywhere. See https://www.gnu.org/software/make/manual/html_node/Echoing.html.

---

I got this error if it is inside a if

---

@echo: command not found

---

so changed in all places except in condition